packages/libs/react-components/src/components/Grid/GridContainer.tsx

```diff
@@ -0,0 +1,70 @@
+import { styled } from './../../styles';
+
+import type * as Stitches from '@stitches/react';
+import React, { FC, ReactNode } from 'react';
+
+export interface IGridContainerProps {
+  children?: ReactNode;
+  spacing?: Stitches.VariantProps<typeof BasicContainer>['spacing'];
+  templateRows?: Stitches.CSSProperties['gridTemplateRows'];
+  templateColumns?: Stitches.CSSProperties['gridTemplateColumns'];
+  templateAreas?: Stitches.CSSProperties['gridTemplateAreas'];
+}
+
+const BasicContainer = styled('div', {
+  display: 'grid',
+  gridTemplateColumns: 'repeat(12, 1fr)',
+  gridGap: '$md',
```

Comment:
This is a matter of preference, but I think it's a bit clearer to define the default gridGap using the default variant

Reply:
You should be able to remove `gridGap: "$md"` now that the default variant is defined

Reply:
not sure if you saw this since the original comment was resolved